share/spack/gitlab/cloud_pipelines/stacks/e4s-cray-sles-cce/spack.yaml

```diff
@@ -0,0 +1,63 @@
+spack:
+  include:
+  - $spack/share/spack/gitlab/cloud_pipelines/configs/linux/ci.yaml
+
+  view: false
+
+  concretizer:
+    reuse: false
+    unify: false
+
+  packages:
+    all:
+      require: '%cce'
+      providers:
```

Comment:
@eugeneswalker would it make sense to just configure these in the cray-sles package.yaml? It seems to be repetitive and the common set of choices when dealing with cray.

Reply:
Right, I was only talking about the providers.